lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -109,14 +109,14 @@ confederation_id:
     config_set_append: '<state> confederation identifier <id>'
 
 confederation_peers:
-  # TODO: Supported in XR but broken as identifier is nvgened on a seperate line.
   default_value: ""
```

Comment:
@glennmatthews Thoughts on adding `kind: string`?  Also, I'm not sure whether we need `auto_default: false` or not